strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -146,7 +146,7 @@
                                                                              "metadata-foo" + ":" +
                                                                              "3.2");
     @Inject
-    private ArtifactEntryService artifactEntryService;
+    private ArtifactRepository artifactEntityRepository;
```

Comment:
It looks like the naming is inconsistent (unless I'm overlooking something). If it's an `ArtifactRepository`, shouldn't it just be `artifactRepository`?

Reply:
renamed

Reply:
Thanks! :)